### PR TITLE
PAYARA-2194 Remove archive name suffix for module names

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

**Comment by jGauravGupta** [comment]
jenkins test please

**Comment by jGauravGupta** [comment]
jenkins test please

File: appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/util/ResourceValidator.java [hunk 1]
```diff
@@ -1036,6 +1037,18 @@ public boolean find(String jndiName, JndiNameEnvironment env) {
                 return nonPortableJndiNames.contains(jndiName);
             }
         }
+       
+        /**
+         * Remove suffix from the module name.
+         */
+        private String getActualModuleName(String moduleName) {
+            if(moduleName != null){
```
Comment (arjantijms) [inline appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/util/ResourceValidator.java]: Tiny small thing, but convention for formatting asks for some extra spaces here:
```java
if (moduleName != null) {
```

**Comment by jGauravGupta** [comment]
jenkins test please

**Pull request merged**